pytest-selenium/pages/content.py

```diff
@@ -61,6 +62,10 @@ def bookbanner(self):
     def toolbar(self):
         return self.ToolBar(self)
 
+    @property
+    def mobile(self):
```

Comment:
this function name could be better

Reply:
and it should probably be `content.toolbar.mobile_search` not `content.mobile_search`

Reply:
I updated the function name as mobile_search. 

But I am not clear about your second comment on making it `content.toolbar.mobile_search`. I made it `content.mobile_search` because the root that I defined for 'MobileSearch' is a separate div that  is a sibling to the Toolbar div and I am not sure if we need to bring MobileSearch under the Toolbar which would probably need changing the root of Toolbar as well.
I am sure what I wrote is very confusing. We can discuss on a call if you want